### PR TITLE
Reformat escaper

### PR DESCRIPTION
**Pull request by yuki0iq**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -34,13 +34,14 @@ def ping(update: Update, context: CallbackContext):
...
-		.replace("@",r"\@").replace("(",r"\(").replace(")",r"\)").replace(">",r"\>")\
-		.replace("<",r"\<").replace("#",r"\#").replace("+",r"\+").replace("-",r"\-")\
-		.replace("=",r"\=").replace("|",r"\|").replace("{",r"\{").replace("}",r"\}")\
-		.replace(".",r"\.").replace("!",r"\!")
+def escape_md(txt: str) -> str:
+	esc = "_*[]`~@()><#+-=|{}.!\\"
+	ans = []
+	for el in txt:
+		if el in esc:
+			ans += ["\\"]
```
Comment (RiedleroD) [inline main.py:42]: oh my god use fucking append for christs sake

File: main.py [hunk 2]
```diff
@@ -34,13 +34,14 @@ def ping(update: Update, context: CallbackContext):
...
 #taken from https://core.telegram.org/bots/api#markdownv2-style
-def escape_md(txt: str) -> str:#TODO: reformat
-	return txt.replace("\\",r"\\").replace("_",r"\_").replace("*",r"\*")\
-		.replace("[",r"\[").replace("]",r"\]").replace("`",r"\`").replace("~",r"\~")\
-		.replace("@",r"\@").replace("(",r"\(").replace(")",r"\)").replace(">",r"\>")\
-		.replace("<",r"\<").replace("#",r"\#").replace("+",r"\+").replace("-",r"\-")\
-		.replace("=",r"\=").replace("|",r"\|").replace("{",r"\{").replace("}",r"\}")\
-		.replace(".",r"\.").replace("!",r"\!")
+def escape_md(txt: str) -> str:
+	esc = "_*[]`~@()><#+-=|{}.!\\"
```
Comment (RiedleroD) [inline main.py:38]: should probably be a global constant

**Review by RiedleroD — CHANGES_REQUESTED** [review]
besides that one comment, definitely a better approach than mine lol

File: main.py (hunk 1 above)
Comment (yuki0iq) [reply main.py]: fixed

File: main.py (hunk 2 above)
Comment (yuki0iq) [reply main.py]: fixed

**Review by RiedleroD — APPROVED** [review]

**Pull request merged**